firmware/STN1110.c

```diff
@@ -29,56 +29,102 @@
 #include <stdlib.h>
 #include "settings.h"
 #include "system.h"
+#include "modp_numtoa.h"
 
 #define LOG_PFX "SYS_STN1110: "
 
-#define STR_HELPER(x) #x
-#define STR(x) STR_HELPER(x)
-#define STN1110_RUNTIME_BAUD_RATE 230400
+#define RESET_DELAY 10
```

Comment:
NIT: Typically I add the suffix `_MS` to indicate these are millisecond values.  It is not necessary here, but it's helpful to others who may not know your code.